puppet.py

```diff
@@ -27,6 +27,22 @@ def check_disabled():
         sudo('test -f {0} && echo DISABLED || echo ENABLED'.format(lockfile))
 
 @task
+def regenerate_certificate():
+    """Regenerate a node certificate"""
+    if len(env.hosts) > 1:
+      abort("This task should only be run on one host at a time.")
+    sudo('rm -f /etc/puppet/ssl/certs/{}.pem'.format(env.hosts[0]))
+    with settings(host_string='puppetmaster-1.management.production'):
+      sudo('puppet cert clean {}'.format(env.hosts[0]))
```

Comment:
As it stands, `govuk_node_clean` isn't open to the world. I don't like including code that others can't see inside repositories like this. The ultimate resolution is to open `govuk_node_clean` up, but since its behaviour can largely be replicated using two `puppet node` commands (which have their source open to the world, and are well-used), I'm not sure that we should open it up. Instead, we should probably deprecate our own use of `govuk_node_clean` in favour of not making our Puppet usage snowflake-y internally.